docs/docs/dev/versions.json

```diff
@@ -0,0 +1,4 @@
+{
+  "helm": ["0.8.0","0.8.1"],
+  "kuma": ["1.5.0-rc2"]
```

Comment:
should this be here?

Reply:
Yeah it's to have a version in the `dev` branch

Reply:
OK, but it doesn't actually show up anywhere if I understood correctly?